how-to/operations/enable-and-disable-maintenance-mode.rst

```diff
@@ -0,0 +1,51 @@
+Enable and Disable Maintenance Mode
+===================================
+
+Overview
+--------
+
+Maintenance mode helps protect the cluster during operations that could be disruptive. It is particularly useful when an operator needs to manually perform tasks on a node that may pose a risk of failure or data loss.
```

Comment:
(Not a native speaker, nit only) `Maintenance mode helps protect the cluster during operations that could be disruptive` -> `Maintenance mode helps protecting the cluster from potentially disruptive maintenance operations `